dist/js/medium-editor.js

```diff
@@ -1020,6 +1020,11 @@ MediumEditor.extensions = {};
             }
         },
 
+        /*
```

Comment:
As for the test file, yes I don't know why the automatic indentation of my editor must have shifted lines which had nothing to do with my changes. I changed the test itself to be more relevant, specific and simple. Thank you for your review.